src/amuse/community/athena/downloadhttp.py

```diff
@@ -4,7 +4,7 @@
 import os
```

Comment:
there is a seperate issue for the dowload scripts..

Reply:
yes - will address that in a separate PR...